### PR TITLE
Of ports and paths

### PR DESCRIPTION
**Pull request by edwardcallahan**
Adds section to Considerations about ports, paths and cnames.

### CODE REVIEW

**Comment by huntc** [comment]
Love it. Thanks.

File: src/main/play-doc/operation/ClusterSetupConsiderations.md [hunk 1]
```diff
@@ -25,3 +25,13 @@ The ability to deploy and manage bundles thus requires the ability to access the
...
+
+# Managing Ports and Paths
+
+It is generally preferred to expose as few ports as required into the cluster from the public network. Ports should be limited to `80` and `443` in general. Most all cluster bundles are thus differentiated on the internet by host name or path. Exceptions to this include internal service ports, e.g. `5444`.  These ports might be used for services only to be used internally, with an internal facing load balancer, that is not exposed to the internet.
+
+We'll also generally use few edge routers. Management of the deployment, from TLS certificates, to DNS and other domain specific functions can provide compelling reason to deploy over multiple edge routers. For example, an ELB for `domain1` and another ELB for `domain2`.
+
+Differentiation of bundle location by path is particularly useful in larger systems wishing to expose fewer hostnames. Developers should note the ability to [preserve paths](CreatingBundles#Preserving-paths-at-the-proxy) at the proxy when applications require such.
+
+Use the `conduct services` command from the [[ConductR CLI|CLI]] to see what host name or port and path currently deployed bundles provide.
```
Comment (huntc) [inline src/main/play-doc/operation/ClusterSetupConsiderations.md:37]: Ah actually -we're deprecating `conduct services` for 2.0. You should assume port 9000 out of the box right? cc. @fsat

Comment (edwardcallahan) [reply src/main/play-doc/operation/ClusterSetupConsiderations.md:37]: Oh. How can we get this info in 2.0?

Comment (huntc) [reply src/main/play-doc/operation/ClusterSetupConsiderations.md:37]: The port 9000 is assumed by default - unless you change that in the HAProxy config. Any further recommendation @fsat ?

**Pull request merged**